README.md

```diff
@@ -5,10 +5,13 @@ ably-python
 
 A Python client library for [www.ably.io](https://www.ably.io), the realtime messaging service.
 
-Works with Python 2.7 and 3.4 onwards.
+## Supported platforms
 
-User support for older Python 3.2 and 3.3 versions is still provided through
```

Comment:
Don't you need to be explicit that we support 3.x as well? Since 2.x and 3.x are so different, it's not necessarily implied by saying 2.7+. 

Again, I think we should link to the `travis.yml` for the set of versions that we test with.

Reply:
We removed 3.3 from Travis when we moved to pytest, because pytest doesn't work with 3.3, but Matthew asked to add that line saying that we still support 3.2 and 3.3

Personally I think 3.2 and 3.3 are ancient versions, unsupported upstream, and barely used. I would stick to the versions defined in Travis, only.

Reply:
OK, so shouldn't this say

    This SDK supports Python 2.7+ and 3.4+.

?

Reply:
Yes

Reply:
👌 updated.

Reply:
Very small detail. There's not, and there won't be, a 2.8 version, Python 2.7 is the last one in the 2.x series. So maybe it should read ``2.7`` instead of ``2.7+``